core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -442,15 +442,17 @@ private[spark] class TaskSchedulerImpl(
     synchronized {
       if (activeExecutorIds.contains(executorId)) {
         val hostPort = executorIdToHost(executorId)
-        logError("Lost executor %s on %s: %s".format(executorId, hostPort, reason))
+        lazy val msg = s"Lost executor $executorId on $hostPort: $reason"
+        if (!sc.isStopped) logError(msg)
```

Comment:
Style is off.

    if (!sc.isStopped) {
      logError(...)
    }

Also the `lazy val msg = ...` sounds overkill here, since the message is only used once. Just inline it.